app/src/main/kotlin/io/eugenethedev/taigamobile/data/repositories/Utils.kt

```diff
@@ -33,7 +33,8 @@ fun CommonTaskResponse.toCommonTask(commonTaskType: CommonTaskType) = CommonTask
     taskType = commonTaskType,
     colors = color?.let { listOf(it) } ?: epics.orEmpty().map { it.color },
     isClosed = is_closed,
-    tags = tags.orEmpty().map { Tag(name = it[0]!!, color = it[1].fixNullColor()) }
+    tags = tags.orEmpty().map { Tag(name = it[0]!!, color = it[1].fixNullColor()) },
```

Comment:
Are you sure it[0] won't be null? I prefer to handle such situations...

Reply:
That's really strange, I mean, why is this even in diff, it's identical (or maybe I'm just blind and can't see the difference). But yes, it is always not null and it has been like this for a while now.